packages/bff/.env

```
@@ -11,3 +11,28 @@ CATALOG_PROCESS_URL="http://localhost:3000"
 ATTRIBUTE_REGISTRY_PROCESS_URL="http://localhost:3000"
 PURPOSE_PROCESS_URL="http://localhost:3000"
 
+TENANT_ALLOWED_ORIGINS="IPA"
+
+ALLOW_LIST_CONTAINER="interop-local-bucket"
+ALLOW_LIST_PATH="allow-list"
+ALLOW_LIST_FILE_NAME="allow-list.txt"
+
+INTERNAL_JWT_KID=ffcc9b5b-4612-49b1-9374-9d203a3834f2
+INTERNAL_JWT_SUBJECT="dev-refactor.interop-backend-for-frontend"
+INTERNAL_JWT_ISSUER="dev-refactor.interop.pagopa.it"
+INTERNAL_JWT_AUDIENCE="refactor.dev.interop.pagopa.it/internal"
+INTERNAL_JWT_SECONDS_DURATION=60
+
+GENERATED_JWT_KID=ffcc9b5b-4612-49b1-9374-9d203a3834f2
+GENERATED_JWT_ISSUER="dev-refactor.interop.pagopa.it"
+GENERATED_JWT_AUDIENCE="refactor.dev.interop.pagopa.it/ui"
+GENERATED_JWT_SECONDS_DURATION=60
```

Could these be renamed into `SESSION_JTW_...` so that it's clearer what they are for?
```suggestion
SESSION_JWT_KID=ffcc9b5b-4612-49b1-9374-9d203a3834f2
SESSION_JWT_ISSUER="dev-refactor.interop.pagopa.it"
SESSION_JWT_AUDIENCE="refactor.dev.interop.pagopa.it/ui"
SESSION_JWT_SECONDS_DURATION=60
```

---

@galales @micdes-pagopa I agree it's a better naming but I need you to confirm I could change it

---

This renaming means that we should also change the platform deployment project.
I agree that it's cleaner, but I would keep the name of these envs unchanged.